src/components/common/navbar.tsx

```diff
@@ -1,10 +1,12 @@
-import { FC, useState } from "react";
+import { FC, KeyboardEvent, useState } from "react";
 import SephoraLogo from "../../assets/logo.svg";
 import MenuIcon from "../../assets/menu.svg";
 import SearchBar from "./searchbar";
 import CustomImage from "./customimage";
 import { useNavigate } from "react-router-dom";
 import { MENU_LIST } from "../utils/Utils";
+import QuickLinksIcon from "../../assets/Quick Links.svg";
```

Comment:
Keep all asset imports together. Move this to line 4.

Reply:
done